src/lib/UI/Config/Provider/FieldType/RichText/AlloyEditor.php

```diff
@@ -44,4 +45,20 @@ protected function getExtraPlugins(): array
 
         return [];
     }
+
+    /**
+     * This will be deprecated in eZ Platform 3.x.
```

Comment:
`This will be removed in eZ Platform 3.0` would be more accurate.

Reply:
Changed the message in https://github.com/ezsystems/ezplatform-admin-ui/pull/943/commits/68aab509aae5470b553822080a5990b1910a7990